grr/server/grr_response_server/gui/ui/store/client_page_facade.ts

```diff
@@ -75,6 +77,15 @@ export class ClientPageStore extends ComponentStore<ClientPageState> {
         };
       });
 
+  // Reducer updating the selected client.
```

Comment:
Please us JSDoc comments `/** ... */` over regular comments `//` when your comment acts as documentation of a public class member.

Reply:
Will update all the similar comments in the file, so there won't be any issues anymore